src/Data/Vec/Relation/Binary/Equality/Setoid.agda

```diff
@@ -12,6 +12,7 @@ module Data.Vec.Relation.Binary.Equality.Setoid
   {a ℓ} (S : Setoid a ℓ) where
 
 open import Data.Nat.Base using (ℕ; zero; suc; _+_)
+import Data.Fin as F
```

Comment:
Seemingly, there's no need for the qualified `import`:
```agda
open import Data.Fin.Base using (Fin; zero; suc)
```
will do just fine, and the overloading of the constructors `zero` and `suc` is harmless here as regards parsing/typechecking.

Reply:
It is not just opened with zero and suc.